catalog/Third-party_APIs.yml

```
@@ -55,3 +55,5 @@ shards:
   description: A library to access the Twitter API
 - github: jaredsmithse/ynab.cr
   description: A library to interact with your YNAB data
+- github: arubinofaux/hunter.cr
+  description: A Crystal-Lang wrapper for Hunter.io API V2
```

Thanks for adding your shard!

Since everything is a Crystal shard, I don't think there's a need to mention that in the description.
```suggestion
  description: A wrapper for Hunter.io API V2
```
Instead it might be worth adding a description for hunter.io. Seems to be a discovery service for email addresses...?

---

Also, do you think the API version needs to be mentioned? Do you not intend to follow up with V3 should that ever be a thing?